consensus/consensus.go

```diff
@@ -332,9 +332,9 @@ func accumulateRewards(
 			continue
 		}
 		numAccounts++
-		account := common.HexToAddress(member.EcdsaAddress)
+		account := member.EcdsaAddress
 		getLogger().Info("rewarding block signer",
-			"account", account,
+			"account", common.HexToAddress(account),
```

Comment:
Actually my bad, we don't need this change.  x_x

```suggestion
			"account", account,
```